recipe/activate/activate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#eval `python "$CONDA_PREFIX"/etc/conda/sage-conda-env.py activate`
+
+export SAGE_ROOT="$CONDA_PREFIX"
+export SAGE_LOCAL="$CONDA_PREFIX"
+
+#source "$CONDA_PREFIX/bin/sage-env"
```

Comment:
Take a look at the gdal scripts, they may help you. But I never 'sourced' in the activation phase.